batch/batch/api.py

```diff
@@ -35,32 +43,51 @@ def create_job(self, url, spec, attributes, batch_id, callback, parent_ids,
         if copy_service_account_name:
             doc['copy_service_account_name'] = copy_service_account_name
 
-        response = requests.post(url + '/jobs/create', json=doc, timeout=self.timeout)
+        response = requests.post(url + '/jobs/create',
```

Comment:
might be easier to have `self.post(...)` with defaults.

Reply:
done